.github/workflows/vale.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: Vale
+on:
+  pull_request:
+    paths:
+      - 'guides/common/*.adoc'
```

Comment:
Is this recursive? I'd be worried this is globbing just on `guides/common/*.adoc` instead of `guides/common/**/*.adoc`.

Reply:
It found errors in a guilds/common/module/<file>.

Reply:
I think this was actually a problem. We can see in https://github.com/theforeman/foreman-documentation/pull/3165 that it's not triggered.

This PR also modified the other trigger (`vale.yml`) so it wasn't a problem (and vale doesn't care about it).

Looking at https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-pull_request-workflow-based-on-files-changed-in-a-pull-request they explicitly use `**.js`.